### PR TITLE
Fix template color head table in email template

### PR DESCRIPTION
**Pull request by agareev**

### CODE REVIEW

File: template/email.html [hunk 1]
```diff
@@ -323,7 +323,12 @@
       <div class="content">
         <table class="main" width="100%" cellpadding="0" cellspacing="0">
           <tr>
+            {{ if gt (len .Alerts.Firing) 0 }}
             <td class="alert alert-warning">
+            {{ end }}
+            {{ if gt (len .Alerts.Resolved) 0 }}
```
Comment (brian-brazil) [inline template/email.html]: Both of these can be true, you want an else here.

**Comment by agareev** [comment]
I've fixed conditionals.

**Comment by agareev** [comment]
@brian-brazil

**Comment by brian-brazil** [comment]
This logic doesn't make sense. Some alerts being resolved does not mean things are okay, as there are still alerts firing.

**Comment by agareev** [comment]
I haven't understand what did you mean (sorry for my bad english).
Isn't my pull request need?

**Comment by mxinden** [comment]
@brian-brazil Could you help @agareev with this PR again?

**Pull request closed without merging**